src/inputs/labeled-inputs.scss

```diff
@@ -41,8 +41,13 @@
       color: t(iui-text-color);
     }
 
-    &::after {
-      content: ':';
+    &.iui-required {
```

Comment:
I would prefer us handling required state (same as disabled, readonly) versus class..

Reply:
Two problems:
- `:required` is set on `input` which comes after `.iui-label`, and there is no easy CSS selector for that.
- We are not using `input` everywhere, so separate class makes sense. Think of LabeledSelect (which is not using native `select`).

Reply:
hmm...
well maybe, then will handle required state in react, I guess..
Now the only concern is that it is a breaking visual change in react too...